### ISSUE TITLE
Frickl Makeover

### ISSUE THREAD

**Issue by Rolf-B** [SELFHTML/SelfTrack#5]
Platzierung von Script-Elementen

**Comment by Rolf-B** [comment]
Aktuell gemeldete Probleme (ohne Bewertung, ob man sie lösen sollte oder nicht):

- Das Frickl-Ergebnis steckt in einer Data-URL und kann deswegen nicht mit den Entwickler-Tools untersucht werden
- Der JS-Teil für das Frickl muss als script-Element im <head> stehen, ansonsten zeigt Frickl es im Body UND Head an und es wird doppelt ausgeführt. Das bedingt, dass Event-Beispiele immer mit DOMContentLoaded Eventhandler laufen müssen
  - jsFiddle hatte mal einen Schalter, wo das User-Script geladen werden sollte. Mittlerweile setzen sie es immer ans Ende des Body
- Es ist nicht möglich, in einem Frickl JS Libraries zu verwenden (z.B. jQuery)
- Das Frickl-Ergebnis verwendet - im Gegensatz zum Vorschau-Tab - nicht das Selfhtml-Grundlayout. Folge: Vorschau und Frickl-Ergebnis können unterschiedlich aussehen.
  - Idee von Felix: Grundlayout per @import hereinholen
- Es ist nicht möglich, ein Frickl-Script als ECMAScript-Modul zu schreiben
- Jürgen: Höhe der Vorschau einstellbar (SetupFrickl in common.js + Beispielvorlage)

**Comment by Matthias-Scharwies** [comment]
Style-Bereiche innerhalb eines svg-Elements überschreiben den style-Bereich im head.

SMIL funktioniert nicht im Firefox - in Chromium-Browsern ist alles ok

**Comment by Rolf-B** [comment]
1. Das ist Working As Designed, würde ich sagen. Welches Todo im Rahmen eines Frickl-Makeover siehst Du hier?
2. Was hat die FF-Kompatibilität bei Frickl zu suchen?_

**Comment by Matthias-Scharwies** [comment]
>1.Das ist Working As Designed, würde ich sagen. Welches Todo im Rahmen eines Frickl-Makeover siehst Du hier?

Ja. Dann muss es aber auch dokumentiert werden, dass wir eben nur minimal-Beispiele einstellen sollten.

Zu 2. Evtl. findet man den Fehler / Bug und kann ihn umgehen. Andererseits passt es jetzt mit dem Warnhinweis im Wiki. SMIL wird eh durch CSS-Animations verdrängt.

**Comment by Matthias-Scharwies** [comment]
>     2. Was hat die FF-Kompatibilität bei Frickl zu suchen?_

Problem ist ja nicht, dass SMIL im FF nicht geht, sondern dass das Frickl im FF SMIL nicht ausführt.

**Comment by Matthias-Scharwies** [comment]
SVG wird vom Frickl gerendert; 

In der Code-Darstellung werden schließende Tags allerdings nicht in einer neuen Zeile, sondern direkt nach dem schließenden Kindelement notiert.
